tensorflow/contrib/slim/python/slim/data/tfexample_decoder_test.py

```diff
@@ -228,7 +228,10 @@ def testDecodeExampleWithJpegEncodingAt16BitCausesError(self):
     image_shape = (2, 3, 3)
     unused_image, serialized_example = self.GenerateImage(
         image_format='jpeg', image_shape=image_shape)
-    with self.assertRaises(TypeError):
+    # decode_raw support uint16 now so ValueError will be thrown instead.
```

Comment:
Can you also add a unit test that tests that decode_raw can now successfully decode uint16?